plugin-examples/src/main/java/com/hotels/styx/ExampleMessageIntercept.java

```diff
@@ -0,0 +1,78 @@
+package com.hotels.styx;
```

Comment:
This is not a valid Java class. I will explain why further down. To avoid submitting invalid classes in pull requests, please attempt to run the associated unit test(s) first.